src/actions/index.js

```diff
@@ -36,3 +35,25 @@ export function updateBeer(id, rating) {
     payload: request,
   };
 }
+
+export function addNewBeer(data, file) {
+  const request = db
+    .collection('Beers')
+    .doc()
+    .set(data)
+    .then(() => {
+      return new Promise(resolve => {
+        return resolve('Success');
+      });
```

Comment:
You can do `return Promise.resolve('Success')` or even shorter:
`.then( () => 'Success' )`